app/ui/src/test/java/com/fsck/k9/ViewHelper.kt

```diff
@@ -0,0 +1,12 @@
+package com.fsck.k9
+
+import android.graphics.drawable.ColorDrawable
+import android.view.View
+import android.widget.TextView
+
```

Comment:
Those are extensions, not helpers.

Reply:
Are you suggesting to rename the file to `ViewExtensions.kt`?

Reply:
Yes, those are extensions.

Reply:
Extension functions can be helpers, too :) But I get the point. I renamed the file to `ViewTestExtensions.kt` to emphasize they're only to be used for testing.